custom/icds_reports/reports/early_initiation_breastfeeding.py

```diff
@@ -203,62 +190,26 @@ def get_early_initiation_breastfeeding_data(domain, config, loc_level, show_test
         in_month = row['in_month']
         name = row['%s_name' % loc_level]
 
-        if tmp_name and name != tmp_name:
-            chart_data['green'].append([tmp_name, (loc_data['green'] / float(rows_for_location or 1))])
-            chart_data['orange'].append([tmp_name, (loc_data['orange'] / float(rows_for_location or 1))])
-            chart_data['red'].append([tmp_name, (loc_data['red'] / float(rows_for_location or 1))])
-            rows_for_location = 0
-            loc_data = {
-                'green': 0,
-                'orange': 0,
-                'red': 0
-            }
-
         birth = row['birth']
 
-        value = (birth or 0) * 100 / float(in_month or 1)
+        value = (birth or 0) / float(in_month or 1)
```

Comment:
Why are these not being multiplied by 100 anymore?

Reply:
Because we use d3.format('.2%') and d3 do this for us.